migen/build/platforms/icebreaker.py

```diff
@@ -0,0 +1,57 @@
+from migen.build.generic_platform import *
+from migen.build.lattice import LatticePlatform
+from migen.build.lattice.programmer import IceStormProgrammer
+
+
+_io = [
+    ("user_ledr_n",   0, Pins("11"), IOStandard("LVCMOS33")),
+    ("user_ledg_n",   0, Pins("34"), IOStandard("LVCMOS33")),
```

Comment:
Other platforms simply number LEDs - what do "r" and "g" stand for?

Reply:
it stands for red and green led's. We have fixed colors no them and it is good to know. Also we have some led that are negative logic and some that are positive. That is what the _n stands for.